lib/fetch_book_data.rb

```diff
@@ -18,8 +18,8 @@ def fetch_titles
     return [] if results.blank?
 
     items = results.fetch('items', [])
-    # All books except for "retired" ones
-    books = items.select{ |i| i['book_state'] != 'retired' }
+    # See https://github.com/openstax/openstax-cms/blob/main/books/constants.py#L7 for book states
+    books = items.select{ |i| i['book_state'] != 'retired' || i['book_state'] != 'unlisted' }
```

Comment:
Shouldn't it be `i['book_state'] != 'retired' && i['book_state'] != 'unlisted'`?

Reply:
I'm trying to make sure it's not shown if it's retired or unlisted (two different states)... wouldn't that only exclude if they are both retired and unlisted, which can't happen .

Reply:
You want `!(A == B || A == C)` which is equivalent to `(A != B && A != C)`
Or you can just use `!['retired', 'unlisted'].include?(i['book_state'])`
Or use `reject` instead of `select` to remove the negation.

Reply:
`(A != B || A != C)` where `B != C` is just `true` all the time.

Reply:
ah, thank you! that would have been confusing (as if I wasn't enough already)... 
`!['retired', 'unlisted'].include?(i['book_state'])` looks like an elegant solution, that's way more readable... I'll go with that